data/mods/Magiclysm/recipes/blacksmithing.json

```
@@ -49,9 +64,10 @@
     "subcategory": "CSC_OTHER_OTHER",
     "skill_used": "spellcraft",
```

```suggestion
    "skill_used": "chemistry",
```
chemistry with alchemy makes way mor sense now that chemistry is a thing.